Source/Methods/RFmxDPD/RFmxDPD.cs

```diff
@@ -0,0 +1,194 @@
+﻿using NationalInstruments.RFmx.SpecAnMX;
+using NationalInstruments.ModularInstruments.NIRfsg;
+using static NationalInstruments.ReferenceDesignLibraries.SG;
+using System;
+
+namespace NationalInstruments.ReferenceDesignLibraries.Methods
+{
+    public static class RFmxDPD
+    {
+        #region Type_Definitionss
+        public struct CommonConfiguration
+        {
+            public double MeasurementInterval_s;                                                 
+            public RFmxSpecAnMXDpdMeasurementSampleRateMode SampleRateMode;
+            public double SampleRate_Hz;
+            public RFmxSpecAnMXDpdSignalType SignalType;
+            public RFmxSpecAnMXDpdPreDpdCfrEnabled PreDpdCFREnabled;
+            public RFmxSpecAnMXDpdApplyDpdCfrEnabled ApplyDpdCFREnabled;
+            public double DutAverageInputPower_dBm;
+                                                            
```

Comment:
I think we should add Sync method too
`specAn.Dpd.Configuration.ConfigureSynchronizationMethod()`